src/board/system76/common/usbpd/tps65987.c

```diff
@@ -25,6 +25,9 @@ static int16_t usbpd_current_limit(void) {
         if (value[0] == 6) {
             uint32_t pdo = ((uint32_t)value[1]) | (((uint32_t)value[2]) << 8) |
                 (((uint32_t)value[3]) << 16) | (((uint32_t)value[4]) << 24);
+            if (!pdo) {
+                return -0x2000;
```

Comment:
What does this value mean?

Also no need braces for single statements

Reply:
No valid PDO was returned.

(Actually -0x2000 is already used below on i2c read failure, so changed this to 0x1000 instead.)

Reply:
Ughh. Can we add some error codes in macros for it?

Reply:
Added an enum for error codes.